README.md

```diff
@@ -755,11 +755,190 @@ Each of the `resource-limits` settings below contain two numeric fields: `hard-l
 Please see the [`getrlimit` linux manpage](https://man7.org/linux/man-pages/man7/capabilities.7.html) for meanings of `hard-limit` and `soft-limit`.
 
 The full list of resource limits that can be configured in Bottlerocket are:
-
-resource limit | setting | default value
------ | ----- | -----
-`RLIMIT_NOFILE` | `settings.oci-defaults.resource-limits.max-open-files.hard-limit` | 1048576
-`RLIMIT_NOFILE` | `settings.oci-defaults.resource-limits.max-open-files.soft-limit` | 65536
+<table>
```

Comment:
It's highly unlikely we will want to render into anything other than HTML, but is there a reason for using HTML tags directly rather than sticking with markdown table syntax?

Reply:
So that we can use rowspan. This will avoid the repetition of `RLIMIT_NOFILE`  rlimit names.